samples/python/cloudevents/txt-to-pdf/README.md

```diff
@@ -5,15 +5,18 @@
 This Python function listens for AWS S3 create events. When a new `.txt` file is detected, it will attempt to convert the text file into a PDF, then upload it to S3.
 
 ## Prerequisites
-* Cloud Native Runtimes: 1.0.0+
+* TriggerMesh
+* Knative Eventing
 * AWS CLI
 * Docker
 * Buildpack CLI
 * Kapp
 * Ytt
 
+> ⚠️ This sample will require working dependencies TriggerMesh and Knative Eventing. If you are using Tanzu Application Platform (TAP), those will be added in TAP 1.4. Using the Cloud Native Runtimes package can replace these two dependencies
```

Comment:
do we know what's needed? like min-versions or something people can use to build their own cluster with Knative and use this sample?